src/XMLimport.cpp

```diff
@@ -907,6 +907,13 @@ void XMLimport::readHost(Host* pHost)
     if (mudlet::self()->mpCurrentActiveHost == pHost) {
         mudlet::self()->dactionInputLine->setChecked(compactInputLine);
     }
+    if (attributes().hasAttribute(QLatin1String("CommandLineHistorySaveSize"))) {
+        pHost->setCommandLineHistorySaveSize(attributes().value(QLatin1String("CommandLineHistorySaveSize")).toInt());
+    } else {
+        // This is the default valuem though prior to the introduction of this
+        // it would have effectively been zero:
+        pHost->setCommandLineHistorySaveSize(50);
```

Comment:
Oh, need to change the original default of 50 to match the 500 in the preferences now...